src/libs/actions/Report.js

```diff
@@ -215,6 +215,7 @@ function getSimplifiedReportObject(report) {
             `lastRead_${currentUserAccountID}`,
             'timestamp',
         ], 0),
+        newMarkerSequenceNumber: getUnreadActionCount(report) === 0 ? 0 : lastReadSequenceNumbers[report.reportID] + 1,
```

Comment:
Could you please help me understand this change a bug?

Reply:
I am still waiting for your answer here. What is this change doing?

Reply:
this change is setting to ONYX the value of `newMarkerSequenceNumber` when we create a report object.
I think it does the same as we were doing before using `isLoadingReportData` ONYXKEY but it is so much simpler, it has the same logic used here on line 389:
https://github.com/Expensify/App/blob/de5d2ba79fa86fa3510009ed618b88e0b1f4c5fa/src/pages/home/report/ReportActionsView.js#L381-L391

once we load a report, we also load its `newMarkerSequenceNumber` that's what it does

Reply:
This feels wrong. Why are you setting `lastReadSequenceNumbers[report.reportID] + 1` when we get new data from server.

Reply:
This is how it should be:
1. we call to fetch all reports. Merged the data asap (You did already). 
2. At this moment, we don't know if we have a new action. We just recalc unreadCount. 
3. Now when the new action is received via fetchActions. ReportactionViews should update the newmarker. 

Now the challenge is that FetchActions is completed before fetchAllReports. 

So the task would be to set the new marker via ReportActionsView when we have received new actions and only if the message is unread and from other users. 

So I think solution might be calling `            this.updateUnreadIndicatorPosition(this.props.report.unreadActionCount);
` after  https://github.com/Expensify/App/blob/3d2ec235a49198fe7759bfff0bf022e644b3d27b/src/pages/home/report/ReportActionsView.js#L231. 

What do you think?

Reply:
I think another simpler approach would be to get the action Report Details with fetchActions call separately from fetchAllReports.

Reply:
Ok, so I think maybe I shouldn't have commited thoses last changes I'll revert it.

Reply:
I'm looking into your suggestted changes...
I'll try to give a feedback till tomorrow

Reply:
Ping me whenever you have some updates for me. Thanks.

Reply:
> So I think solution might be calling this.updateUnreadIndicatorPosition(this.props.report.unreadActionCount); after

That's not gonna work, if we do that, the value of `this.props.report.unreadActionCount` won't be the updated value, we only will have the right value of `unreadActionCount` after `fetchAllReports`, by your solution we receive a new message from `fetchActions`, but we won't update `unreadActionCount`, so we will have a wrong value of `unreadActionCount`.

> So the task would be to set the new marker via ReportActionsView when we have received new actions and only if the message is unread and from other users.

How you think we could do that? As fetchActions will only give us the pending messages from a report but won't give which of thoses messages are new and therefore won't update `unreadActionCount`.

Reply:
> I think another simpler approach would be to get the action Report Details with fetchActions call separately from fetchAllReports.

What you mean with this?

Reply:
> What do you mean by this? doesn't it already works separately within ReportActionsView.js?

Code only pulls the actions in the ReportActionView.js's `didMount`. I was suggesting getting the details as well. But I think that is by design this would need to be discussed on approved first.

Looking into other comment...